package.json

```diff
@@ -24,6 +24,14 @@
     "bundle:prod": "cross-env NODE_ENV=production webpack --config webpack.config.js",
     "docs": "typedoc --readme none ."
   },
+  "prettier": {
```

Comment:
Why did you leave the other `prettier` configurations in the package.jsons?

Reply:
I don't remember, but it looks like it'll be fine without it. I'll remove them.